<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionDialog</class>
 <widget class="QDialog" name="OptionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OptionDialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>251</width>
     <height>321</height>
    </rect>
   </property>
   <property name="title">
    <string>Parameters for linkage</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Link width:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditLinkWidth">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>130</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Link depth:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditLinkDepth">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Joint radius:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditJointRadius">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>190</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Joint cap radius1:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Joint cap radius 2:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Joint cap depth:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Hole radius:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditHoleRadius">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>170</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Slider guide width:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Slider guide depth:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditJointCapRadius1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>210</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditJointCapRadius2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>230</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditJointCapDepth">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>250</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSliderGuideWidth">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>270</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSliderGuideDepth">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>290</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Gap:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditGap">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>110</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonLarge">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Large</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSmall">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Small</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonTwice">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>x 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonHalf">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>x 0.5</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Body margin:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditBodyMargin">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonOK">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>400</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Parameters for body</string>
   </property>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Body depth:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditBodyDepth">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>pushButtonOK</tabstop>
  <tabstop>pushButtonCancel</tabstop>
  <tabstop>pushButtonSmall</tabstop>
  <tabstop>pushButtonLarge</tabstop>
  <tabstop>lineEditGap</tabstop>
  <tabstop>lineEditLinkWidth</tabstop>
  <tabstop>lineEditLinkDepth</tabstop>
  <tabstop>lineEditHoleRadius</tabstop>
  <tabstop>lineEditJointRadius</tabstop>
  <tabstop>lineEditJointCapRadius1</tabstop>
  <tabstop>lineEditJointCapRadius2</tabstop>
  <tabstop>lineEditJointCapDepth</tabstop>
  <tabstop>lineEditSliderGuideWidth</tabstop>
  <tabstop>lineEditSliderGuideDepth</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
